test/Dialect/FIRRTL/lower-types.fir

```diff
@@ -21,13 +21,6 @@ circuit TypeCanonicalizationConsideredHarmful:
     ; CHECK-NEXT: firrtl.partialconnect %a_a_a, %ax_a_a
     ; CHECK-NEXT: firrtl.partialconnect %a_a_a, %ax_a_a
 
-    a <= ax
-    a.a <= ax.a
-    a.a.a <= ax.a.a
-    ; CHECK:      firrtl.connect %a_a_a, %ax_a_a
-    ; CHECK-NEXT: firrtl.connect %a_a_a, %ax_a_a
-    ; CHECK-NEXT: firrtl.connect %a_a_a, %ax_a_a
```

Comment:
FYI: the tests in this file are not removed. They are migrated into `lower-types.mlir` as they now work. 🎊